assets/app/actions/siteActions.js

```diff
@@ -77,21 +77,35 @@ export default {
     }).catch(error => alertActions.alertError(error.message));
   },
 
+  createPR(site, head, base) {
+    return github.createPullRequest(site, head, base).then((pr) => {
+      return github.mergePullRequest(site, pr);
+    }).then(() => {
+      return github.deleteBranch(site, head);
+    }).then(() => {
+      this.fetchBranches(site);
+    }).then(() => {
+      return alertActions.alertSuccess(`${head} merged successfully`);
+    }).catch(error => alertActions.httpError(error.message));
```

Comment:
Does it make sense to have any error recovery if any of these operations fail? I'm thinking there's nothing to really recover from (the PR got merged, but the branch couldn't be deleted for whatever reason so we'd need to manually address it later) but am happy to entertain options.